pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -200,7 +197,11 @@ func createSlirpNetwork(iface v1.Interface, network v1.Network, domain *api.Doma
 	return nil
 }
 
-func CalculateNetworkQueues(vmi *v1.VirtualMachineInstance) uint32 {
+func CalculateNetworkQueues(vmi *v1.VirtualMachineInstance, ifaceType string) uint32 {
+	if !isMultiqueue(vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue) || ifaceType != "virtio" {
```

Comment:
Can we get `"virtio"` from some central place (types?).

Reply:
virtio is all over, 
added ModelVirtio which will be used for interface / device model virtio, and also for bus / disk / gfx model virtio
(didn't create define a specific for each entity)

will push soon